Upstream-Status: Pending

--- a/lv_drv_conf_template.h	2022-05-11 18:50:39.848654645 +0000
+++ b/lv_drv_conf_template.h	2022-05-11 18:40:25.650469080 +0000
@@ -306,7 +306,7 @@
  *  Linux frame buffer device (/dev/fbx)
  *-----------------------------------------*/
 #ifndef USE_FBDEV
-#  define USE_FBDEV           0
+#  define USE_FBDEV           1
 #endif
 
 #if USE_FBDEV
@@ -306,7 +306,7 @@
  * Mouse or touchpad as evdev interface (for Linux based systems)
  *------------------------------------------------*/
 #ifndef USE_EVDEV
-#  define USE_EVDEV           0
+#  define USE_EVDEV           1
 #endif
 
 #ifndef USE_BSD_EVDEV
