From 52657f512e55f73d5a91fc2d6d7d1986e1aaad1b Mon Sep 17 00:00:00 2001
From: Andrei Gherzan <andrei@gherzan.ro>
Date: Wed, 22 May 2019 18:32:07 +0100
Subject: [PATCH] Allow enable DRI without DRI drivers

Upstream-Status: Pending
Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

---
 meson.build       | 2 +-
 meson_options.txt | 6 ++++++
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index b4cd7999566..f65a6392913 100644
--- a/meson.build
+++ b/meson.build
@@ -154,7 +154,7 @@ with_dri_r200 = dri_drivers.contains('r200')
 with_dri_nouveau = dri_drivers.contains('nouveau')
 with_dri_swrast = dri_drivers.contains('swrast')
 
-with_dri = dri_drivers.length() != 0 and dri_drivers != ['']
+with_dri = get_option('dri') or (dri_drivers.length() != 0 and dri_drivers != [''])
 
 gallium_drivers = get_option('gallium-drivers')
 if gallium_drivers.contains('auto')
diff --git a/meson_options.txt b/meson_options.txt
index 3ab2900f929..abbc91f3ef0 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -35,6 +35,12 @@ option(
   choices : ['auto', 'true', 'false'],
   description : 'enable support for dri3'
 )
+option(
+  'dri',
+  type : 'boolean',
+  value : false,
+  description : 'enable support for dri'
+)
 option(
   'dri-drivers',
   type : 'array',
