From 5e7efc5e1615c65c8c17457355f552ef65af7508 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 18 May 2019 08:33:53 +0000
Subject: [PATCH] lima_screen: add PIPE_TEXTURE_CUBE

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>

---
 src/gallium/drivers/lima/lima_screen.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gallium/drivers/lima/lima_screen.c b/src/gallium/drivers/lima/lima_screen.c
index 5e6ac1ffb08..7649631a131 100644
--- a/src/gallium/drivers/lima/lima_screen.c
+++ b/src/gallium/drivers/lima/lima_screen.c
@@ -263,6 +263,7 @@ lima_screen_is_format_supported(struct pipe_screen *pscreen,
    case PIPE_BUFFER:
    case PIPE_TEXTURE_1D:
    case PIPE_TEXTURE_2D:
+   case PIPE_TEXTURE_CUBE:
       break;
    default:
       return false;
