From f679f528fb7f792cd821b0c68f108baeb1478569 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 18 May 2019 08:33:53 +0000
Subject: [PATCH] lima_screen: add PIPE_TEXTURE_CUBE

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>

---
 src/gallium/drivers/lima/lima_screen.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gallium/drivers/lima/lima_screen.c b/src/gallium/drivers/lima/lima_screen.c
index cd9122672b6..30b199e777f 100644
--- a/src/gallium/drivers/lima/lima_screen.c
+++ b/src/gallium/drivers/lima/lima_screen.c
@@ -273,6 +273,7 @@ lima_screen_is_format_supported(struct pipe_screen *pscreen,
    case PIPE_TEXTURE_1D:
    case PIPE_TEXTURE_2D:
    case PIPE_TEXTURE_RECT:
+   case PIPE_TEXTURE_CUBE:
       break;
    default:
       return false;
